tests/test_items.cpp

```diff
@@ -950,6 +950,33 @@ TEST_F(ItemTest, CultureProperty)
     task = service().get_task(item_id);
     EXPECT_EQ("yo-NG", task.get_culture());
 }
+
+TEST_F(ItemTest, CreateItemsWithVector)
+{
+    std::vector<ews::contact> contacts;
+    for (int i = 0; i < 10; i++)
+    {
+        contacts.push_back(ews::contact());
+    }
+
+    auto contact_ids = service().create_item(contacts);
+    EXPECT_EQ(10u, contact_ids.size());
+}
```

Comment:
Well behaved tests should remove the items they create at the end. Use `ews::internal::on_scope_exit` for this.